package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "mali-error-mapper",
+  "version": "0.0.1",
+  "description": "Error mapper for Mali applications",
+  "keywords": [
+    "error",
+    "grpc",
+    "mali",
+    "mapper"
+  ],
+  "homepage": "https://github.com/uphold/mali-error-mapper#readme",
+  "bugs": {
+    "url": "https://github.com/uphold/mali-error-mapper/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/uphold/mali-error-mapper.git"
+  },
+  "license": "MIT",
+  "author": "Uphold",
+  "main": "src/index.js",
+  "scripts": {
+    "changelog": "uphold-scripts changelog $npm_package_version",
+    "lint": "uphold-scripts lint",
+    "release": "uphold-scripts release",
+    "test": "uphold-scripts test",
+    "version": "uphold-scripts version"
+  },
+  "pre-commit": [
+    "lint"
+  ],
+  "jest": {
+    "modulePaths": [
+      "<rootDir>"
+    ]
+  },
+  "dependencies": {
+    "@uphold/http-errors": "git+ssh://git@github.com/uphold/http-errors#v1.2.0",
```

Comment:
Should this be a peer dependency? Otherwise code like `if (!(e instanceof HttpError)) {` wouldn't behave as intended if the project using this were using a different version of `http-errors`, as it would have a different version of `HttpError` that wouldn't match the one in this package.

Probably the same for `grpc-error`.

Reply:
Will those as peer dependencies.

Reply:
Still need to move `@uphold/http-errors` and `grpc-error` to  `devDependencies`, so that they don't get used in projects using this package.